### DISCUSSION TITLE
Transfer a row/topic/to-do item from one Outline to another Outline

### DISCUSSION THREAD

**Discussion by soritchie** [vincode-io/Zavala#228] [Ideas]
My use case for this is that I generally start out with a list of things I want to do. Create row, add notes etc. with more detail.

And then one of those rows actuallly turns out to be its own project, with multiple things that need done. So I create a new Outline to manage that new project.

But moving the items/rows and related notes from the original 'Ideas' outline to the new outine isn't trivial on iOS or MacOS. Copy/Paste isn't the best experience.

I'd prefer the new 'transferred' item to be at the top of the list of items on the existing Outline I sent it to, and  use existing functionality to move it where I want.

Thanks so much for this software. I use it every day.

**Comment by vincode-io** [comment]
I plan at some point adding a function like "Extract" that you could use on a Row. It would make a new Outline with that Row's Topic as the new Title and the Row's children as the top level Rows. Is that what you are asking for?

Could you also elaborate on what you don't like about the Cut/Copy/Paste functionality? The problems with it may seem obvious to you, but I didn't know there is a usability issue. Please provide as much detail as you can.

Thanks for using Zavala and providing feedback. You would be surprised at how useful information like this is to me.

**Reply by soritchie** [reply]
Hello! Yes, the extract as described above would work great for me. 

In the past, when I copy and paste a row it on my iPhone it dumps the pasted row in raw markdown, and doesn’t preserve note content as note content. 

However, following your question above, I now realise there are 2 paste options. One option ( a short press on the new blank row) is probably the default iOS behaviour. 

But a long press on the new blank row does a different paste operation and everything is pasted correctly. 

That wasn’t obvious:)

Now that I know this I’m happy to withdraw my request. I can work with what is already available. 

Thanks for your reply.

**Reply by poembox** [reply]
Exactly! Thanks very much! I thought I was crazy being forced to paste on to a bullet. Why would anyone want to paste markdown when they just copied an outline?
